Makefile

```diff
@@ -22,21 +22,24 @@ ARCH_ALIAS_aarch64 = arm64
 ARCH_ALIAS = $(shell echo "$(ARCH_ALIAS_$(ARCH))")
 
 NERDCTL_VERSION=1.0.0
+OPENRESTY_VERSION=0.0.1
 QEMU_VERSION=v6.2.0
 CRI_DOCKERD_VERSION=0.2.3
 CRI_DOCKERD_ORG=Mirantis
 BINFMT_IMAGE=tonistiigi/binfmt:qemu-$(QEMU_VERSION)
 
 .PHONY: mkimage
-mkimage:
+mkimage: openresty-v$(OPENRESTY_VERSION)-$(ARCH)
 	cd src/aports && git fetch && git checkout $(GIT_TAG)
+	cp openresty-v$(OPENRESTY_VERSION)-$(ARCH) openresty.tar
```

Comment:
Would it make more sense to pass `OPENRESTY_VERSION` in as a `build-arg`? Then we can just `ADD openresty-v$(OPENRESTY_VERSION)-$(ARCH) /home/build/packages/lima/openresty.tar` in the Dockerfile instead.

Reply:
I didn't do it because I wasn't sure what a reasonable default value would be..

It also means we have to pass `ARCH` as well, but maybe that isn't a bad idea anyways.